### PR TITLE
Update item-denied-member email contents

### PR DESCRIPTION
**Pull request by Baggerone**
image

image

### CODE REVIEW

**Review by Schparky — APPROVED** [review]
I think the item card needs more indention and part of it needs a background color. Look at the claim card. Specifically, add a `div` wrapper around the ID, description, and category.
```
<div style="background-color: #fafbfc; padding: 16px;">
 ...
</div>
```
and then another div on the rest of it, but only the `padding: 16px;`

File: application/templates/mail/_item_card.plush.html [hunk 1]
```diff
@@ -38,13 +38,13 @@ <h4 style="margin:0; padding:0;">Unique ID</h4>
 
 <%= partial("button", {
 	url: itemURL,
-	label: "Open Item in " + appName}
+	label: buttonLabel}
```
[the remark this answers is not in the document]
Comment (Baggerone) [reply application/templates/mail/_item_card.plush.html:41]: The Figma designs have different label text for different item based emails.

[the remark this answers is not in the document]
Comment (Schparky) [reply application/templates/mail/_item_card.plush.html:41]: Even if you kept it in the item card, you could still construct the text in the top level of the template and pass it into the component. No need for another property inserted into the template.

**Comment by Baggerone** [comment]
I'll get the div style changes done in the next PR. (I forgot to switch back to this repo to make those changes, but I did manually test a denied item email with them in.)

**Pull request merged**